include/clang/Interpreter/CppInterOp.h

```diff
@@ -465,7 +483,7 @@ namespace Cpp {
                                                 TCppIndex_t param_index);
 
   ///\returns function that performs operation op on lc and rc
-  TCppFunction_t GetBinaryOperator(TCppScope_t scope, const std::string& op,
+  TCppFunction_t GetBinaryOperator(TCppScope_t scope, enum BinaryOperator op,
```

Comment:
We don't need an elaborated type here. Dropping the enum should be fine.

Reply:
I was thinking of removing our definition of the enum and replacing it with `using clang::BinaryOperatorKind;`. Then we will not require that long `switch ... case ...`  in `CppInterOp.cpp::GetBinaryOperator`. Nor any kind of string comparison.
Any thoughts?

Reply:
We can't leak implementation details for clang. That means we can't include header files of llvm/clang in CppInterOp.h.